### PR TITLE
Made some changes

### PR DESCRIPTION
**Pull request by gui1612**
Changes that have been done:

- Changed README.md

- Commented the code to make it easier to navigate on it

- Implemented a regular spam option (without the guild stuff) that may or may not be working :)))

### CODE REVIEW

File: home_cinema_bot.py [hunk 1]
```diff
@@ -120,6 +140,25 @@ async def spam_emote(ctx, emote_name: str = "no emote", num_times: int = -1):
...
+async def spam(ctx, spam_input: str, num_times: int = -1):
+    print("Starting the spam:")
+    the_message: discord.Message = ctx.message
+
+    if(the_message):
+        await the_message.delete()
+        
+    for i in range(num_times):
+            spam = spam_input
+            await ctx.send(str(spam))
```
Comment (Naapperas) [inline home_cinema_bot.py:155]: No need to cast to str since input already comes as str

File: home_cinema_bot.py [hunk 2]
```diff
@@ -120,6 +140,25 @@ async def spam_emote(ctx, emote_name: str = "no emote", num_times: int = -1):
...
+
+@bot.command(name="spam", help="Starts spamming whatever you want")
+async def spam(ctx, spam_input: str, num_times: int = -1):
+    print("Starting the spam:")
+    the_message: discord.Message = ctx.message
+
+    if(the_message):
+        await the_message.delete()
+        
+    for i in range(num_times):
```
Comment (Naapperas) [inline home_cinema_bot.py:153]: Does range(-1) go to infinity ?

File: home_cinema_bot.py [hunk 3]
```diff
@@ -23,27 +29,37 @@
...
 spamming_emotes = {}
 
-emotes = ["kekw", "pepega", "sadge", "peepohappy", "monkaW", "ELIMINAR",
+
+#Supported emotes 
+
+emotes = ["kekw", "pepega", "sadge", "peepohappy", "monkaW", "ELIMINAR",                 
           "pufavo", "pogchamp", "weirdchamp", "homies", "FeelsStrongMan",
-          "5head", "bruh", "caragomeu", "ehehe", "mds", "facho"]
+          "5head", "bruh", "caragomeu", "ehehe", "mds", "facho", "Exkirbyrainbow"]
```
Comment (Naapperas) [inline home_cinema_bot.py]: Delet kirby

File: home_cinema_bot.py (hunk 1 above)
Comment (Naapperas) [reply home_cinema_bot.py:155]: Also, unnecessary lines of code. You just need to send the argument passed in.

Comment (gui1612) [reply home_cinema_bot.py:155]: Yup, seemed to be like that, just copypasted the code lines from #emote_spam tho

File: home_cinema_bot.py (hunk 2 above)
Comment (gui1612) [reply home_cinema_bot.py:153]: Also copypasta from emote_spam

File: home_cinema_bot.py (hunk 3 above)
Comment (gui1612) [reply home_cinema_bot.py]: It's outdated, the newer version already has it fixed

**Pull request closed without merging**